config/deploy/production.rb

```diff
@@ -57,7 +56,7 @@
 #     auth_methods: %w(publickey password)
 #     # password: "please use keys"
 #   }
-server "heroes-team.pp.ua",
+server "ec2-52-57-126-232.eu-central-1.compute.amazonaws.com",
```

Comment:
in future, in PR make changes only related to you task, any other changes should be placed in other PR with appropriate description